ipatests/prci_definitions/nightly_ipa-4-9_previous.yaml

```diff
@@ -535,6 +535,19 @@ jobs:
         timeout: 10800
         topology: *master_1repl
 
+  fedora-previous-ipa-4-9/test_installation_TestInstallWithoutSudo:
+    requires: [fedora-previous-ipa-4-9/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{fedora-previous-ipa-4-9/build_url}'
+        update_packages: True
```

Comment:
update_packages should not be set for 4-9_previous.

Reply:
Fixed, thanks for the catch.